src/Dmishh/Bundle/SettingsBundle/Entity/SettingOwner.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Dmishh\Bundle\SettingsBundle\Entity;
+
+/**
+ * This interface must be implemented by the Entity connected to a setting
+ */
+interface SettingOwner
+{
+    public function getUsername();
```

Comment:
You could question if this is the proper name for this function. A better name would be `getIdentifier` or `getSettingsIdentifier`. I let it be `getUsername` because I believed it would be easier to update... But I might be wrong.